jest.config.js

```diff
@@ -7,12 +7,6 @@ const config = {
 };
 /*! m0-end */
 
-process.env.DCS_JWT_SECRET = '123';
-process.env.DCS_LOG_LEVEL = 'silent';
-process.env.NOTIFY_API_KEY = '123apiKey';
-process.env.WEBHOOK_DEV_REPORTER_SLACK = 'webhook-url';
-process.env.APP_ENV = 'test';
```

Comment:
Are none of these being used at all in tests?

Reply:
Closed as per: https://github.com/CriminalInjuriesCompensationAuthority/data-capture-service/pull/236#discussion_r757049178